src/tests/krate/publish/basics.rs

```diff
@@ -22,7 +22,7 @@ fn new_krate() {
     assert!(crates[0].deps.is_empty());
     assert_eq!(
         crates[0].cksum,
-        "8a8d84b87f379d5e32566b14df153c0ab0e1ea87dae79a00b891bb41f93dbbf6"
+        "270bbe1624abd766746bf9938b791fadd88e7e0135339510837e11b45e167350"
```

Comment:
This has changed because the `PublishBuilder` by default sets a description and license on the published packages. Previously those two fields haven't been written to the generated `Cargo.toml` file, but now they are, which caused the tarball checksum to change.